### PR TITLE
Warn user that safari does not support touch events.

### PR DESCRIPTION
**Pull request by David-Orangemoon**
Warn the user that Safari does not support touch events, and only Google supports speech recognition.

Exactly what it says on the tin it also adds the <Supports Touches?> block just so we can check if a device supports touch events.

Apple is a pain.
Screenshot 2023-05-24 8 50 09 AM
Screenshot 2023-05-24 8 50 02 AM
Screenshot 2023-05-24 8 49 53 AM
Just got done with my Spanish exam and I'm already having to make this PR. This is why I don't dev for Apple most of the time lol.

### CODE REVIEW

**Comment by David-Orangemoon** [comment]
I also got rid of the hacky fix in favor of just warning the user.

File: extensions/obviousAlexC/SensingPlus.js [hunk 1]
```diff
@@ -286,22 +276,19 @@
...
-    userAgent.includes("iPhone") ||
-    userAgent.includes("iPod") ||
-    userAgent.includes("iPad")
-  ) {
-    //* <-- Its a problem with all IOS browsers from what I see.
-    hackyTouchFix = true;
-  }
-  if (
+  let supportsTouches = true;
+  if (userAgent.includes("Safari")) {
```
[the remark this answers is not in the document]
Comment (David-Orangemoon) [reply extensions/obviousAlexC/SensingPlus.js]: I've remedied the now by the use of regex.

**Comment by TheShovel** [comment]
> Apple is a pain.
So true.

**Comment by Skyhigh173** [comment]
does chrome (in iOS) support touch event?

**Comment by David-Orangemoon** [comment]
> does chrome (in iOS) support touch event?

Sadly no because chrome on iOS is based on Safari.

**Comment by GarboMuffin** [comment]
the reason I've been stalling on this is because the proper solution is to make it work, because it most definitely is possible

**Comment by GarboMuffin** [comment]
this will do for now

**Pull request merged**